shared/src/business/useCases/public/casePublicSearchInteractor.test.js

```diff
@@ -23,127 +23,6 @@ describe('casePublicSearchInteractor', () => {
     expect(results).toEqual([]);
   });
 
-  it('should filter out sealed cases', async () => {
```

Comment:
I'm assuming these tests will return in some form before this issue is closed, but want to check with you on that.

Reply:
Sealed cases are still returned, but they don't have any more detail than the docket number.